tests/scripts/remote_monitoring_tests/rest_apis/test_list_recommendations.py

```diff
@@ -1444,3 +1444,249 @@ def test_invalid_list_recommendations_notification_codes(cluster_type: str):
 
         response = delete_experiment(json_file)
         print("delete exp = ", response.status_code)
+
+@pytest.mark.sanity
+def test_list_recommendations_cpu_mem_optimised(cluster_type: str):
```

Comment:
Add test description here and also update the md file with the test cases added

Reply:
@chandrams I have added the test description and updated the readme.